### PR TITLE
SW-4968 Remove 'Projects' feature flag and baked in truthy result of the flag

### PR DESCRIPTION
**Pull request by nickgraz**

### CODE REVIEW

**Comment by nickgraz** [comment]
* **#2277** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2277?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @nickgraz and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/components/Reports/Router.tsx [hunk 1]
```diff
@@ -19,7 +17,7 @@ const ReportsRouter = (): JSX.Element | null => {
   return (
     <Switch>
       <Route exact path={APP_PATHS.REPORTS}>
-        {featureFlagProjects ? <ReportsView tab={'reports'} /> : <ReportList />}
+        <ReportsView tab={'reports'} /> : <ReportList />
```
Comment (karthikbtf) [inline src/components/Reports/Router.tsx:20]: remove ReportList?

**Review by karthikbtf — COMMENTED** [review]
Thanks 🙏 
One comment that may need a fix.

File: src/components/Reports/Router.tsx (hunk 1 above)
Comment (karthikbtf) [reply src/components/Reports/Router.tsx]: it is outside of an evaluation `{..}` and considered raw jsx.. the ':' would be considered a string maybe

File: playwright/e2e/suites/accession.spec.ts [hunk 2]
```diff
@@ -120,7 +120,7 @@ export default function AccessionTests() {
 
     await page.getByRole('button', { name: 'Seeds' }).click();
     await page.getByRole('button', { name: 'Accessions' }).click();
-    await page.locator('#row1-estimatedWeightGrams').getByText('495').click();
+    await page.locator('#row1-accessionNumber').getByText('24-1-2-001').click();
```
Comment (karthikbtf) [inline playwright/e2e/suites/accession.spec.ts:123]: 🤞

**Review by karthikbtf — APPROVED** [review]
Hoping this works 🙏

**Pull request merged**